### PR TITLE
Preparations for release 1.6.0 [v2]

### PR DESCRIPTION
**Pull request by clebergnu**
Cleanups and CI fixes in preparation for release 1.6.0

---

Changes from v1 (#63):
 * Removed `subprocess` conditional import relevant to Python 2 only (@luckyh)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -73,7 +73,7 @@ rpm-release: srpm-release
 	mock -r $(MOCK_CONFIG) --resultdir BUILD/RPM -D "rel_build 1" --rebuild BUILD/SRPM/python-aexpect-$(VERSION)-*.src.rpm
 
 check: clean
-	inspekt checkall --disable-lint W1618,R0205
+	inspekt checkall --disable-lint W1618,R0205,W0707,R1725
```
Comment (pevogam) [inline Makefile:76]: @clebergnu Will write this down in the issue you created but we can always prioritize fixing more than the pylint disable cases mentioned there.

**Review by pevogam — APPROVED** [review]

File: aexpect/client.py [hunk 2]
```diff
@@ -52,11 +53,6 @@
 from aexpect.utils import path as utils_path
 from aexpect.utils import wait as utils_wait
 
-if sys.version_info[0] < 3:
```
Comment (ldoktor) [inline aexpect/client.py:55]: There is one more place to be treated:

```diff
diff --git a/aexpect/client.py b/aexpect/client.py
index 1b01c0c..b6bccfc 100644
--- a/aexpect/client.py
+++ b/aexpect/client.py
@@ -319,9 +319,8 @@ class Spawn(object):
         """
         try:
             with open(self.output_filename, 'rb') as output_file:
-                errors = ('replace' if sys.version_info[0] < 3 else
-                          'backslashreplace')
-                return output_file.read().decode(self.encoding, errors)
+                return output_file.read().decode(self.encoding,
+                                                 'backslashreplace')
         except IOError:
             return None
```

**Review by ldoktor — APPROVED** [review]
OK, there is one more py2-related place (well I bet there are many but one more I spotted) but apart from that I think it can be merged. I'll find some time and try to address some of the newly skipped lints.

Update: What I'm trying to say it seems to work well, CI is passing so please force-push the last remaining issue so we can merge and release and I'll take a look at the exception handling later or next week.

**Comment by clebergnu** [comment]
OK, CI passed. Merging with the additional fix found/proposed by @ldoktor

**Pull request merged**